### PR TITLE
feat: get live data

### PR DESCRIPTION
**Pull request by yeshev**
part of #3133

### CODE REVIEW

File: frontend/src/constants/liveTail.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const LIVE_TAIL_HEARTBEAT_TIMEOUT = 10 * 60 * 1000;
```
Comment (YounixM) [inline frontend/src/constants/liveTail.ts:1]: Comment the logic around LIVE_TAIL_HEARTBEAT_TIMEOUT .

Can directly use `600000` in place of `10 * 60 * 1000`.

File: frontend/src/container/LiveLogs/LiveLogsListChart/index.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+		PANEL_TYPES.TIME_SERIES,
+
+		{
+			enabled: isConnectionOpen,
+			refetchInterval: LIVE_TAIL_GRAPH_INTERVAL,
+		},
+		{ dataSource: DataSource.LOGS },
+	);
+
+	const chartData: QueryData[] = useMemo(() => {
```
Comment (YounixM) [inline frontend/src/container/LiveLogs/LiveLogsListChart/index.tsx:47]: I think useMemo isn't required here. We aren't computing anything

File: frontend/src/container/LiveLogs/LiveLogsListChart/index.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+		if (!data) return [];
+
+		return data.payload.data.result;
+	}, [data]);
+
+	const isLoading = useMemo(
```
Comment (YounixM) [inline frontend/src/container/LiveLogs/LiveLogsListChart/index.tsx:53]: Same comment as above

File: frontend/src/container/LiveLogsTopNav/index.tsx [hunk 4]
```diff
@@ -1,30 +1,51 @@
...
+	onOpenConnection,
+}: LiveLogsTopNavProps): JSX.Element {
+	const {
 		isConnectionOpen,
 		isConnectionLoading,
+		initialLoading,
+		handleCloseConnection,
+	} = useEventSource();
+
+	const isPlaying = useMemo(
```
Comment (YounixM) [inline frontend/src/container/LiveLogsTopNav/index.tsx:19]: Same comment as above

File: frontend/src/container/LogsExplorerChart/index.tsx [hunk 5]
```diff
@@ -2,25 +2,29 @@ import Graph from 'components/Graph';
...
-	) => ({
-		label: allLabels[index],
-		data: element,
-		backgroundColor: colors[index % colors.length] || 'red',
-		borderColor: colors[index % colors.length] || 'red',
-	});
+	const handleCreateDatasets: Required<GetChartDataProps>['createDataset'] = useCallback(
+		(element, index, allLabels) => ({
+			data: element,
+			backgroundColor: colors[index % colors.length] || 'red',
```
Comment (YounixM) [inline frontend/src/container/LogsExplorerChart/index.tsx:18]: In place of `red`, we should be using the hex from our figma color 

@kaushikmacharla07  :  Please help with the red color hex we want to use across the product.

File: frontend/src/container/LiveLogs/LiveLogsListChart/index.tsx (hunk 2 above)
Comment (yeshev) [reply frontend/src/container/LiveLogs/LiveLogsListChart/index.tsx:47]: @YounixM We use it like props with memo component. Also, we have context which causes rerender, so useMemo is a good solution here and below examples. If we don't have computing, it doesn't mean we shouldn't use useMemo.

File: frontend/src/container/LiveLogs/LiveLogsContainer/index.tsx [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+				{initialLoading ? (
+					<Col span={24}>
+						<Spinner style={{ height: 'auto' }} />
+					</Col>
+				) : (
+					<>
+						<Col span={24}>
+							<LiveLogsListChart />
+						</Col>
+						test
```
Comment (YounixM) [inline frontend/src/container/LiveLogs/LiveLogsContainer/index.tsx:151]: Remove `test`

**Review by palashgdev — APPROVED** [review]

**Pull request merged**